### ISSUE TITLE
Multiplicative group generators

### ISSUE THREAD

**Issue by kamen-u-cesty** [Czechbol/mathcrypto#1]
I think there is an error in the logic of computing element orders.
```py
>>> import mathcrypto
>>> g = mathcrypto.MultiplicativeGroup(7)
>>> g.get_element_order(2)
6
```
however from
```py
>>> for i in range(g.order):
>>>     print(2**i % g.mod)
1
2
4
1
2
4
```
we can see that 2's order is just three, not six.

**Comment by kamen-u-cesty** [comment]
```py
>>> import mathcrypto
>>> mathcrypto.MathFunctions.factorize(6)
6
```
clearly doesn't work, 6 = 2*3. This function doesn't work for any numbers under 10.

**Comment by czechbol** [comment]
Resolved by e2c3019ffc52304d22126f7b935607ac548d1ea3
